Client/Frontend/Library/HistoryPanel.swift

```diff
@@ -542,20 +542,6 @@ class HistoryPanel: SiteTableViewController, LibraryPanel {
         }
     }
 
-    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
```

Comment:
This class had both viewForHeaderInSection and titleForHeaderInSection implemented identically, just use the latter because it is simpler.